src/versions/saveVersion.ts

```diff
@@ -79,10 +79,46 @@ export const saveVersion = async ({
       const data: Record<string, unknown> = {
         autosave: Boolean(autosave),
         version: versionData,
+        latest: true,
         createdAt: doc?.createdAt ? new Date(doc.createdAt).toISOString() : now,
         updatedAt: draft ? now : new Date(doc.updatedAt).toISOString(),
       };
-      if (collection) data.parent = id;
+
+      if (collection) {
+        data.parent = id;
+
+        await VersionModel.updateMany({
+          $and: [
+            // TODO: is this _id necessary? Is this when you publish from a version?
```

Comment:
I am unsure if this `_id` check is necessary? If not, then it should be removed from 2.0 as well.